test/dockerfiles/mongo/docker-entrypoint-initdb.d-7.0/mongo-init.js

```diff
@@ -0,0 +1,14 @@
+print("Adding travis username to ee2 database for mongo 7.0")
+db = db.getSiblingDB('ee2')
```

Comment:
Why we don't need `db.auth('travis', 'travis')` on mongo7

Reply:
Correct, it causes the init script to fail on mongo7

Reply:
Otherwise, I would be able to use the same script for mongo 3.6 and mongo 7, but I cannot. That's the whole point of using 2 different scripts.

Reply:
👍